tests/functional/test_functionality.c

```diff
@@ -18,9 +18,10 @@
 #include "poison.h"
 
 void
-destroy_test_splinter_shadow_array(test_splinter_shadow_array *sharr)
+destroy_test_splinter_shadow_array(platform_heap_id            hid,
```

Comment:
I think you can get rid of the `hid` argument.

Reply:
Done. Deleted.